FWCore/Catalog/test/FileLocator_t.cpp

```diff
@@ -2,50 +2,52 @@
 #include "catch2/catch_all.hpp"
 
 #include <array>
+#include <memory>
 #include <string>
+#include <vector>
 
 #include "FWCore/Catalog/interface/FileLocator.h"
 #include "FWCore/Catalog/interface/SiteLocalConfig.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
 #include "FWCore/ServiceRegistry/interface/ServiceRegistry.h"
 #include "FWCore/ServiceRegistry/interface/ServiceToken.h"
 
 #include "TestSiteLocalConfig.h"
 
+namespace {
+  std::unique_ptr<edm::FileLocator> makeFileLocator(edm::ServiceToken const& token, unsigned int iCatalog) {
+    edm::ServiceRegistry::Operate operate(token);
+    edm::Service<edm::SiteLocalConfig> localconfservice;
+    auto const& dataCatalogs = localconfservice->dataCatalogs();
+    edm::CatalogAttributes catalogAttributes = dataCatalogs.at(iCatalog);
```

Comment:
Mirroring my other question, can this be
```suggestion
    edm::CatalogAttributes const& catalogAttributes = dataCatalogs.at(iCatalog);
```
?